spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -1152,8 +1152,31 @@ If the participant has already responded successfully to an `@Compensate`
 or `@Complete` method invocation then it MAY report `410 Gone`
 HTTP status code or in the case of non-JAX-RS method returning
 <<source-ParticipantStatus,ParticipantStatus>> `null`.
-This enables the participant to free up resources if it
-hasn't done so previously.
+
+Let's provide some intuition on this matter with the following example.
+
+Let's have an LRA participant annotated with `@Compensate` and `@Status`.
+The LRA is cancelled and the LRA implementation tries to invoke the participant's `@Compensate` method.
+At that time a network failure happens. The LRA implementation receives a network error as the result of the call.
+Now it cannot be sure if the `@Compensate` call pass through and participant has already compensated the work
+and only the response was lost. Or if the `@Compensate` call has not reached the participant at all.
+As the participant defines the `@Status` method the LRA implementation will invoke it to find the participant's true status
+(invocation can be processed several times until succeeds).
+The participant's `@Status` method is expected to indicates whether the `@Compensate` was successfully processed.
+
+* If the participant processed the `@Compensate` call it may <<jaxrs-participant-methods,forget about the LRA identification>> already.
```

Comment:
```suggestion
* If the participant processed the `@Compensate` call it may <<jaxrs-participant-methods, has already forgotten about the LRA identification>>.
```